lib/store_web/live/backoffice/order_live/show.ex

```diff
@@ -0,0 +1,59 @@
+defmodule StoreWeb.Backoffice.OrderLive.Show do
+  use StoreWeb, :live_view
+
+  import Ecto.UUID
+  import Ecto.Query
+  import Store.Inventory
+  alias Store.Repo
+  alias Store.Inventory
+  alias Store.Inventory.Order
+  alias Store.Inventory.OrdersProducts
+  alias Store.Uploaders
+  alias Store.Accounts
+
+  @impl true
+  def mount(%{"id" => id}, _session, socket) do
+    {:ok, assign(socket, order: Inventory.get_order!(id))}
+  end
+
+  @impl true
+  def handle_params(%{"id" => id}, _, socket) do
+    {:noreply,
+     socket
+     |> assign(:current_page, :orders)
+     |> assign(:page_title, page_title(socket.assigns.live_action))
+     |> assign(:order, Inventory.get_order!(id) |> Repo.preload(:products))}
+  end
+
+  @impl true
+  def handle_event("paid", _payload, socket) do
+    order = socket.assigns.order
+
+    order
+    |> Order.changeset(%{status: :paid})
+    |> Repo.update!()
```

Comment:
I think it would be a good idea to refactor this code into a function in `Inventory.ex` like so:

```elixir
def change_order_status(order, status) do
  order
  |> Order.changeset(%{status: ^status})
  |> Repo.update!()
end
 ```